src/Spec-Core.package/ComposablePresenter.class/class/platform.st

```diff
@@ -0,0 +1,4 @@
+platform
+platform
+
+	^ SpecInterpreter bindings
```

Comment:
Spec adapters start to have a behavior of platform (provide Spec platform specific code)